packages/client/src/vscode/services.ts

```diff
@@ -105,21 +97,19 @@ export const initServices = async (instruction: InitServicesInstruction) => {
  *   - languages
  *   - model
  */
-export const importAllServices = async (instruction: InitServicesInstruction) => {
-    const lc: InitializeServiceConfig = instruction.serviceConfig ?? {};
-    const userServices: monaco.editor.IEditorOverrideServices = lc.userServices ?? {};
+export const importAllServices = async (instructions: InitServicesInstructions) => {
+    const userServices: monaco.editor.IEditorOverrideServices = instructions.userServices ?? {};
 
     const lcRequiredServices = await supplyRequiredServices();
 
     mergeServices(lcRequiredServices, userServices);
-    await configureExtHostWorker(instruction.serviceConfig?.enableExtHostWorker === true, userServices);
+    await configureExtHostWorker(instructions.enableExtHostWorker === true, userServices);
 
-    reportServiceLoading(userServices, instruction.logger);
+    reportServiceLoading(userServices, instructions.logger);
 
-    if (instruction.performChecks === undefined || instruction.performChecks()) {
-        await initialize(userServices, undefined, lc.workspaceConfig);
-        const logLevel = lc.workspaceConfig?.developmentOptions?.logLevel ?? LogLevel.Info;
-        StandaloneServices.get(ILogService).setLevel(logLevel);
+    if (instructions.performChecks === undefined || (typeof instructions.performChecks === 'function' && instructions.performChecks())) {
+        await initialize(userServices, undefined, instructions.workspaceConfig);
+        StandaloneServices.get(ILogService).setLevel(instructions.logger?.getLevel() ?? LogLevel.Off);
```

Comment:
FYI, you can set the log level [this way](https://github.com/CodinGame/monaco-vscode-api/blob/61752f5a8e77b1359c8648f8455d048d483c7e79/demo/src/setup.common.ts#L241-L243)

Reply:
Yes thanks, I need to updated the config if it is not present. I want to have this visible on higher level in the wrapperConfig. I will adapt the code.

Reply:
Done ⬇️